subprojects/docs/src/samples/groovy/multi-project-with-spock-tests/README.adoc

```diff
@@ -22,22 +22,8 @@ Running the tests:
 ----
 $ ./gradlew test
 
-> Task :application:test
-WARNING: An illegal reflective access operation has occurred
-WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/Users/daniel/.gradle/caches/modules-2/files-2.1/org.codehaus.groovy/groovy/2.5.7/99907efe4b69f800c42584386f5d668e4d952bd5/groovy-2.5.7.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
-WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
-WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
-WARNING: All illegal access operations will be denied in a future release
-
-> Task :library:test
-WARNING: An illegal reflective access operation has occurred
-WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/Users/daniel/.gradle/caches/modules-2/files-2.1/org.codehaus.groovy/groovy/2.5.7/99907efe4b69f800c42584386f5d668e4d952bd5/groovy-2.5.7.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
-WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
-WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
-WARNING: All illegal access operations will be denied in a future release
-
 BUILD SUCCESSFUL
-7 actionable tasks: 7 executed
+9 actionable tasks: 9 executed
```

Comment:
⭕ Maybe we should include the the sample test `.out` file like in other places.

Reply:
I think that does not work currently. It's also the reason why we get this message (on `master`) at the moment I believe:

```
SEVERE: composite_builds.adoc: line 52: include file not found: /Users/jendrik/repos/gradle/subprojects/docs/build/working/usermanual/samples/build-organization/composite-builds/basic/tests/basicCli.out
Aug 12, 2020 7:38:49 PM <script> convertFile
SEVERE: composite_builds.adoc: line 52: include file not found: /Users/jendrik/repos/gradle/subprojects/docs/build/working/usermanual/samples/build-organization/composite-builds/basic/tests/basicCli.out
```

Our tooling looks for the files in the _working_ dir where the samples are copied. But the `test` folder is not part of the sample itself.

I think we should fix this, but it's a separate topic.

Reply:
IIRC we use it at a whole bunch of places. I can look that up tomorrow.

Reply:
That's a regression, but will work after https://github.com/gradle/gradle/pull/14124 is merged.

Reply:
Thanks. I rebased this branch to latest master that includes the fix.

I am still not sure how to do this in the sample docs now. Somehow the samples path is still wrong. If I do:

```
include::{samplesPath}/groovy/multi-project-with-spock-tests/tests/testTask.out[]
```

I get this:
```
SEVERE: sample_groovy_multi_project_with_spock_tests.adoc: line 34: include file not found:
/Users/jendrik/repos/gradle/subprojects/docs/build/working/samples/docs/samples/groovy/multi-project-with-spock-tests/tests/testTask.out
```

Reply:
Follow up: https://github.com/gradle/gradle/issues/14131